src/test/java/org/folio/search/utils/CallNumberUtilsTest.java

```diff
@@ -133,6 +133,18 @@ void excludeIrrelevantResultItems(String callNumberType, List<CallNumberBrowseIt
     assertThat(unchangedItems).isEqualTo(given);
   }
 
+  @Test
+  public void excludeIrrelevantResultItems_with_null_iecnc() {
+    var context = BrowseContext.builder().build();
+    var givenItems = createItemWithNullEffectiveCallNumberComponents();
+    var callNumberTypeValue = "Z669.R360 197";
```

Comment:
"callNumberTypeValue" is supposed to be a call number type name, such as "dewey"/"lc"...